packages/trader/src/Modules/Account/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -0,0 +1,175 @@
+// import PropTypes        from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { convertDateFormat } from '@deriv/shared/utils/date';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import DataTable from 'App/Components/Elements/DataTable';
+import Loading from '../../../../../templates/app/components/loading.jsx';
+import LoadErrorMessage from '../../ErrorMessages/LoadErrorMessage';
+
+const Td = ({ title, text, className }) => (
+    <td>
+        <h3 className='cell-title'>{title}</h3>
+        <span className={classNames('cell-value', className)}>{text}</span>
+    </td>
+);
+
+const LoginHistoryListView = ({ fields, login_history }) => (
+    <table className='login-history-table'>
+        <tbody>
+            <tr>
+                <Td title={fields[0].value} text={login_history[fields[0].key]} />
+            </tr>
+            <tr>
+                <Td title={fields[2].value} text={login_history[fields[2].key]} />
+                <Td className={fields[1].key} title={fields[1].value} text={login_history[fields[1].key]} />
+            </tr>
+            <tr>
+                <Td title={fields[3].value} text={login_history[fields[3].key]} />
+                <Td title={fields[4].value} text={login_history[fields[4].key]} />
+            </tr>
+        </tbody>
+    </table>
+);
+
+const parseUA = user_agent => {
+    const lookup = [
+        { name: 'Edge', regex: /(?:edge|edga|edgios|edg)\/([\d\w.-]+)/i },
+        { name: 'SeaMonkey', regex: /seamonkey\/([\d\w.-]+)/i },
+        { name: 'Opera', regex: /(?:opera|opr)\/([\d\w.-]+)/i },
+        { name: 'Chromium', regex: /(?:chromium|crios)\/([\d\w.-]+)/i },
+        { name: 'Chrome', regex: /chrome\/([\d\w.-]+)/i },
+        { name: 'Safari', regex: /version\/([\d\w.-]+)/i },
+        { name: 'IE', regex: /msie\s([\d.]+[\d])/i },
+        { name: 'IE', regex: /trident\/\d+\.\d+;.*[rv:]+(\d+\.\d)/i },
+        { name: 'Firefox', regex: /firefox\/([\d\w.-]+)/i },
+        { name: 'Binary app', regex: /binary\.com V([\d.]+)/i },
+    ];
+    for (let i = 0; i < lookup.length; i++) {
+        const info = lookup[i];
+        const match = user_agent.match(info.regex);
+        if (match !== null) {
+            return {
+                name: info.name,
+                version: match[1],
+            };
+        }
+    }
+    return null;
+};
+
+class LoginHistory extends React.Component {
+    // data keys+values
+    fields = [
```

Comment:
Can we refactor this to an object?
Something like: 
```
fields = {
  date: localize('Date and time'),
   //...
}
```
The referencing on line 102 and below makes this hard to follow.